jobtalk/urls.py

```diff
@@ -16,11 +16,15 @@
 from django.conf.urls import url
 from django.contrib import admin
 
-from core.views import UserRegistration, successful, ReviewSubmission
+from core.views import (UserRegistration, successful, ReviewSubmission,
+    ReviewList, ReviewDetail)
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
     url(r'^users/', UserRegistration.as_view(), name='user-registration'),
-    url(r'^reviews/', ReviewSubmission.as_view(), name='review-submission'),
+    url(r'^reviews$', ReviewList.as_view(), name='review-list'),
+    url(r'^reviews/$', ReviewList.as_view(), name='review-list'),
+    url(r'^reviews/submit$', ReviewSubmission.as_view(), name='review-submission'),
```

Comment:
I was trying to make `jobtalk.com/reviews` and `jobtalk.com/reviews/` go to the same ReviewList view. I don't know enough regex to do it in one line :) Do you think we should remove `jobtalk.com/reviews` from the url list?